src/main/java/org/craftercms/engine/search/SiteAwareElasticsearchService.java

```diff
@@ -53,4 +73,23 @@ protected void updateIndex(final SearchRequest request) {
         }
     }
 
+    @Override
+    protected void updateFilters(final SearchRequest request) {
+        super.updateFilters(request);
```

Comment:
Were are the other filters for expired and disabled?

Reply:
those are handled by the super call, it has a list of string queries (the same ones we used in solr)